specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -6979,6 +7021,72 @@
       "required": [
         "enabled"
       ]
+    },
+    "GuardrailsProfile": {
+      "type": "object",
+      "properties": {
+        "systemExcludedNamespaces": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of namespaces specified by AKS to be excluded from Guardrails",
+          "items": {
+            "type": "string"
+          }
+        },
+        "version": {
+          "type": "string",
+          "description": "The version of constraints to use"
+        },
+        "level": {
+          "type": "string",
+          "enum": [
+            "Off",
+            "Warning",
+            "Enforcement"
+          ],
+          "x-ms-enum": {
+            "name": "level",
+            "modelAsString": true
+          },
+          "description": "The guardrails level to be used"
```

Comment:
Maybe also include here: "By default, Guardrails is enabled for all namespaces except those that AKS excludes via systemExcludedNamespaces"?

Reply:
Added